guacamole/src/main/frontend/src/app/home/directives/guacRecentConnections.js

```diff
@@ -50,13 +50,24 @@ angular.module('home').directive('guacRecentConnections', [function guacRecentCo
 
             // Required services
             var guacHistory       = $injector.get('guacHistory');
+            var preferenceService = $injector.get('preferenceService');
 
             /**
              * Array of all known and visible recently-used connections.
              *
              * @type RecentConnection[]
              */
             $scope.recentConnections = [];
+            
+            /**
+             * Returns whether or not recent connections should be displayed.
+             * 
+             * @returns {Boolean}
+             *     true if recent connections should be displayed, otherwise false.
+             */
+            $scope.showRecentConnections = function showRecentConnections() {
```

Comment:
I think this should be renamed to avoid confusion, as `showRecentConnections()` sounds like a function that shows the recent connections section, not a function that returns whether recent connections are shown.

Reply:
Renamed via rebase.